code/game/machinery/computer/cloning.dm

```diff
@@ -1,3 +1,5 @@
+var/list/clonecomputers = list()
```

Comment:
This isn't really needed; the `machines` list is small enough to search through that we really don't need a specific list for clone computers.